datahub-web-react/src/app/shared/entitySearch/RelatedEntity.tsx

```diff
@@ -42,10 +38,10 @@ export default ({ searchResult, entityPath }: Props) => {
                 dataSource={entitiesToShow}
                 renderItem={(item) => {
                     return (
-                        <ListItem>
+                        <>
                             {entityRegistry.renderPreview(entityType, PreviewType.PREVIEW, item.entity)}
                             <Divider />
-                        </ListItem>
+                        </>
```

Comment:
how sure are we that this won't affect other screens? Could this mess w/ the formatting of list items in search, browse, user profile page, etc?

Reply:
![image](https://user-images.githubusercontent.com/80503855/148443554-2af2b6a2-8b0b-4caa-ab37-e26da77d7f3b.png)

this are the component which are using RelatedEntity.ts directly or indirectly
Verified for all of them